### PR TITLE
[ADD] inventory: locations dashboard

### PR DESCRIPTION
**Pull request by Felicious**
## Summary

1. Overview of the Locations dashboard, explaining the columns and buttons.
2. Examples of common reports that can be generated from the out-of-the-box filters.

[Task](https://www.odoo.com/odoo/project/3835/tasks/3946500?cids=3)

### CODE REVIEW

**Comment by Felicious** [comment]
Hi @larm-odoo :heart_eyes: 

Tagging you for review on this ~150 line PR! Excited for your feedback :heart_hands:

**Review by larm-odoo — APPROVED** [review]
Hi @Felicious - this was a very interesting doc, I loved the examples - and this gives me ideas for my own future reporting docs!

I only had some suggestions for clarifying places, adding some doc links that may be helpful, and just a quesiton on the chicken example. I am sure you will make the edits needed, so I am approving now!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+The *Locations* dashboard is the go-to place to view on-hand storage locations for company products.
+Use this report to see where stock is stored, identify stranded inventory, or view past inventory to
+see product locations on specific dates.
+
+.. note::
+   The *Reporting* menu in *Inventory* is only accessible to users with :doc:`admin access
+   <../../../../general/users/access_rights>`.
+
+To access the locations report, go to :menuselection:`Inventory app --> Reporting --> Locations`.
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:15]: Amazing catch, Lara!! thank you!!!!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
+===================
+Locations dashboard
+===================
+
+.. |down| replace:: :icon:`fa-caret-down` :guilabel:`(caret down)`
+
+The *Locations* dashboard is the go-to place to view on-hand storage locations for company products.
+Use this report to see where stock is stored, identify stranded inventory, or view past inventory to
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:8]: yes , i believe it's an industry term!
image

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 3]
```diff
@@ -0,0 +1,144 @@
+===================
+Locations dashboard
+===================
+
+.. |down| replace:: :icon:`fa-caret-down` :guilabel:`(caret down)`
+
+The *Locations* dashboard is the go-to place to view on-hand storage locations for company products.
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:7]: I like the idea of removing marketing language from docs. (:

[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:7]:
```suggestion
Use the *Locations* dashboard to view on-hand storage locations for company products.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst (hunk 1 above)
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:15]:
```suggestion
TODO: add setup instructions to enable storage locations in Odoo!!
To access the locations report, go to :menuselection:`Inventory app --> Reporting --> Locations`.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+Click the buttons to the right of each row item to access additional information:
+
+- :icon:`fa-history` :guilabel:`History`: access the stock move history of the product, displaying
+  information about the quantity and description of why the product was moved from one location to
+  another.
+- :icon:`fa-refresh` :guilabel:`Replenishment`: access the :doc:`reordering rules
+  <../../product_management/product_replenishment/reordering_rules>` page to replenish products at
+  the specific location.
+
+Click the the :guilabel:`New` button to make an inventory adjustment to record quantities of a
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:55]:
```suggestion
TODO: add ref link to count_products doc
Click the the :guilabel:`New` button to make an inventory adjustment to record quantities of a
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+      :align: center
+      :alt: Show search result for the location.
+
+The report now displays a list of products at the transit location.
+
+.. example::
+   Searching `Input` in :guilabel:`Location` shows a list of products at the *Wh/Input* location.
+
+   The list shows `500` quantities of `Chicken`, which is alarming if not refrigerated soon after
+   reception. The stranded inventory report helps identify items idling in non-storage locations.
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:119]: Thank you for helping me think of irl use cases. Maybe the chicken wasn't the best example of stranded inventory. I currently don't have other ideas, though...

**Comment by Felicious** [comment]
Hello @odoo/us-doc-review ! This ~ 150 line PR is ready for review (: Excited to hear what you think about the use cases :blush:

**Review by ksc-odoo — APPROVED** [review]
Hey, @Felicious -- just finished my Final Review. Awesome job. Great work on the realistic use cases. Approving now. There is only a handful of relatively minor suggestions that require your attention, so once you implement the necessary changes, feel free to tag this for Tech Review. Thanks! 👍

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+================================
+
+By default, the :guilabel:`Locations` dashboard that lists all on-hand products in stock, along with
+the following information:
+
+- :guilabel:`Location`: current storage location. If a product is stored at `Shelf 1` and `Shelf 2`,
+  the product is listed twice, showing quantities at each location.
+- :guilabel:`Package`: the package product is stored in, if any.
+- :guilabel:`Lot/Serial Number`: if the product has a lot or serial number, it is specified here.
+- :guilabel:`On Hand Quantity`: current quantity of products. Click the :icon:`fa-pencil`
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:33]: Just checked a runbot, and I think the Owner column was a custom column created by someone!  I didn't see it in mine. 

image

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+Dead stock report
+-----------------
+
+To get list of expired items, also referred to as *dead stock*, follow these steps:
+
+#. Go to :menuselection:`Inventory app --> Reporting --> Locations`.
+#. Then, click the |down| icon in the :guilabel:`Search...` bar to reveal a drop-down list of
+   :guilabel:`Filters`, :guilabel:`Group By`, and :guilabel:`Favorite` options.
+#. Choose the :guilabel:`Expiration Alerts` option under the :guilabel:`Filters` section.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:86]:
```suggestion
#. Enable the :guilabel:`Internal Locations` and :guilabel:`Expiration Alerts` option under the :guilabel:`Filters` section.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+A few common reports that can be created using the :guilabel:`Locations` dashboard are detailed
+below.
+
+Dead stock report
+-----------------
+
+To get list of expired items, also referred to as *dead stock*, follow these steps:
+
+#. Go to :menuselection:`Inventory app --> Reporting --> Locations`.
+#. Then, click the |down| icon in the :guilabel:`Search...` bar to reveal a drop-down list of
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:84]: Great point about the internal locations filter! adding that in

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+      :alt: Show items stored at a specific location.
+
+Inventory discrepancy report
+----------------------------
+
+To generate a report of items that have been moved since the last :doc:`inventory audit
+<../inventory_management/cycle_counts>`, follow these steps:
+
+#. Go to :menuselection:`Inventory app --> Reporting --> Locations`.
+#. Then, click the |down| icon in the :guilabel:`Search...` bar to reveal a drop-down list of
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:139]: thank you for the suggestion, it's a really good one!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+   :guilabel:`Filters`, :guilabel:`Group By`, and :guilabel:`Favorite` options.
+#. Choose the :guilabel:`Conflicts` option from the :guilabel:`Filters` section.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/locations.rst:141]:
```suggestion
#. Enable the :guilabel:`Internal Locations` and :guilabel:`Conflicts` option from the :guilabel:`Filters` section.
```

**Comment by Felicious** [comment]
Hello @samueljlieber! This PR is ready for your tech review

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious! Awesome job on this brand new Locations dashboard Inventory reporting doc! Your PR looks good to me, approving with a single suggestion! Thanks for your work :)
.....
@robodoo delegate=Felicious

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**